compsets/all.input

```diff
@@ -1,7 +1,9 @@
 load 'fv3mom6cice5.input'
+load 'fv3mom6cice5_ccpp.input'
```

Comment:
I think that by re-loading the cpl_defaults in your fv3mom6cice5_ccpp.input, you've changed the default INPUT_NML for all tests to the ccpp version. I do not think we want the cpl_defaults in this file.

Reply:
Hmm interesting. Wondering how this is working - the cpl_defaults in the CCPP version change the default INPUT_NML to the CCPP version `input.mom6_ccpp.nml.IN`, which should make the IPD version abort (because it contains additional namelist parameters that IPD doesn't know about). The benchmark tests do overwrite INPUT_NML for each test, but the older tests don't. Confused.

Reply:
For the benchmark tests we have a special benchmark input file because we actually want to keep the settings fixed and not changeable by providing an option (eg @[SATMEDMF]). Is there a chance you had the load fv3mom6cice5_ccpp.input commented out when you ran the IPD tests? Do the directories still exist where you ran the RTs?

Reply:
They were both on definitely, because I was running all tests in one go. I'll kick them off again.

Reply:
It is probably too late to reply, but the simplest solution I think is to remove the cpl_defaults from the ccpp input file and instead place the definition of INPUT_NML in each ccpp test. The SUITE_NAME could remain in the usual cpl_defaults since it would be ignored by the IPD tests.

Reply:
I am about to find out how/why this works in the current code. We need to remember that all this IPD-CCPP coexistence is temporary. The easiest would be to move the load `fv3mom6cice5_ccpp.input` to after the IPD/before the CCPP tests for this commit.

Reply:
Yes, I understand. I am in the same place with the CMEPS tests, which for now are a special case of the NEMS mediator tests.

Reply:
I still don't understand the magic, but it is all correct:
```
[Dom.Heinzeller@hfe06 cpld_fv3_384_mom6_cice_cold_atm_flux_repro]$ pwd
/scratch1/BMC/gmtb/Dom.Heinzeller/FV3_RT/rtgen.193663/tmp/cpld_fv3_384_mom6_cice_cold_atm_flux_repro
[Dom.Heinzeller@hfe06 cpld_fv3_384_mom6_cice_cold_atm_flux_repro]$ diff input.nml ../cpld_fv3_ccpp_384_mom6_cice_cold_atm_flux_repro/input.nml
16a17
>        ccpp_suite = 'FV3_GFS_2017_coupled'
182a184,185
>        oz_phys        = .true.
>        oz_phys_2015   = .false.
290d292
<
```

Reply:
I think I now understand why I thought this would be a problem but (as you've shown) it is not. Your tests rely on separate cpld_fv3_ccpp_mom6_cice_(cold,2d)_atm_flux.input "base" files whereas for CMEPS, we just re-purpose the existing tests. All the ccpp tests reside solely w/in the fv3mom6cice_ccpp.input file and use only the cpl_defaults defined there.

Reply:
I was going to write the same just now, thought about it while running. So, can we accept this for the PR or do we need to change it?

Reply:
I don't think we don't need to change it for the PR. Sorry for the extra trouble.